.github/workflows/e2e_tests.yml

```diff
@@ -73,9 +81,14 @@ jobs:
 
       - name: set up go 1.15
         uses: actions/setup-go@v1
+        if: ${{ !contains(github.event.ref, 'fips') }}
         with:
           go-version: 1.15
 
+      - name: setup fips compliant go
```

Comment:
Is this needed? Do the tests fail without this? If so, we need to know the list of tests failing as it is likely to translate into istioctl not being compatible with fips istio.

Reply:
This got removed, I think you are checking an old revision